Rajitha/itemPrice.js

```diff
@@ -0,0 +1,36 @@
+/**********************************************************************
+ * Create a function that takes an array of objects (groceries) which 
+ * calculates the total price and returns it as a number. A grocery 
+ * object has a product, a quantity and a price,
+ ***********************************************************************/
+const getItemPrice = arr => {
+  let iArr = arr;
+  let itemPrice = 0;
+ 
+  for (let i = 0; i < iArr.length; i++){
+    itemPrice += Number(iArr[i].price) * (iArr[i].quantity);
+  }
+  return itemPrice
+}
+
+const testGetItemPrice = (arr,expected) => {
+  return(getItemPrice(arr) === expected) ? true : false
```

Comment:
I think we went over this.
Next time ill just say Fundamentals and not go over you code till you fix it.

I know i'm mean. 😄 

what's the difference between
```
(getItemPrice(arr) === expected) ? true : false
```
and 
```
getItemPrice(arr) === expected
```
?

Reply:
Also this is a test function so it should show a console.log or some sort of print out to see if the test passed.